.github/workflows/ignore_symlinks.yml

```
@@ -0,0 +1,72 @@
+
+jobs:
+  ignore_symlinks:
+    name: ignore_symlinks
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+    steps:
+    - name: Checkout
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)